roles/sdkman/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Dependencies
+  ansible.builtin.apt:
```

Comment:
I see you like fully-qualified-module names? :) 
Any reason for that? (Apart from ansible's own recommendation of course, which IMO is a bit weird way to solve the underlying problem of naming conflicts)

From my end, there are two main reasons I *don't* like it:
- it messes up module documentation lookup from intellij
- it just looks weird when we write yaml like java :D

Reply:
Apart from "I like it because it is very concise", I think the important reason for start moving to FQMNs is that the ansible community is moving that way. Yes, in the transition from 2.9 to 2.10 it was decided to not break backwards compatibility and include "translations" for the bare names used prior to 2.10. This backward compatibility promise is of course only relevant for stuff that existed in a pre-FQMN world. Any new modules/stuff will not have bare names, forcing us to use a mix. It might also be decided to do a breaking change at some point, so I say lets go all in!

Reply:
Right, makes sense if that's where things are headed.